### PR TITLE
Adding Leena to the Website

### PR DESCRIPTION
**Pull request by BatoolMM**
This is a simple PR to add @LAlmehlisy to the project details - it also adds more details to the description of the project!

### CODE REVIEW

File: _data/people.yaml [hunk 1]
```diff
@@ -225,6 +225,21 @@ alexandra-holinski:
   last-name: Holinski
   orcid: 0000-0003-2321-5568
   pronouns: She/her
+LAlmehlisy:
```
Comment (bebatut) [inline _data/people.yaml:228]: Could you move Leena below in the list? We try to use alphabetical order on the username :)

Comment (bebatut) [reply _data/people.yaml:228]: Can you also put the username in lowercase? Thanks a lot

**Review by bebatut — COMMENTED** [review]
Thanks @BatoolMM. I just made 2 comments to try to keep the same order in `people.yaml`

File: _data/people.yaml (hunk 1 above)
Comment (BatoolMM) [reply _data/people.yaml]: I moved it below with other usernames starting with L

File: _data/people.yaml [hunk 2]
```diff
@@ -2654,6 +2654,21 @@ lauracarter:
   pronouns: she/her
   twitter: LauraC_rter
   website: https://lauracarter.github.io
+lalmehlisy:
```
Comment (bebatut) [inline _data/people.yaml:2657]: Could you move it between kswhitney and landimi2? Thank you :)

Comment (BatoolMM) [reply _data/people.yaml]: Done! let me know if you need any other changes!

File: _data/ols-4-projects.yaml [hunk 3]
```diff
@@ -350,6 +356,7 @@
   participants:
   - mai-alajaji
   - batoolmm
+  - LAlmehlisy
```
Comment (bebatut) [inline _data/ols-4-projects.yaml:359]:
```suggestion
  - lalmehlisy
```

**Comment by bebatut** [comment]
Looks good. Thanks a lot @BatoolMM

**Pull request merged**